### DISCUSSION TITLE
Mount drive, I'm confused, need help.

### DISCUSSION THREAD

**Discussion by cedrik72** [photoprism/photoprism#1290] [Q&A] [answered]
Hi, 

My photos are located on a windows 2012 server, I cant run Docker on it.
I use a NUC PC with windows10 to run Docker and PhotoPrism.
A physical drive is shared from the server and I mounted it as a virtual drive in the NUC, the drive is P:/
It's also \\servcbad\Photos.
I mounted the share in Docker, as several ways (see attached)
Now I tried many (really many, I'm on it for 2 days) ways to write it in my docker-compose.yml without success.
It's working well when the photos are on local drive of the NUC, but no way from the shared disc.
I run Docker on Hyper-V (not WSL 2)

![Capture](https://user-images.githubusercontent.com/6243418/118150430-d3c13e00-b412-11eb-9472-43d7e9cb9a6e.JPG)
![Capture2](https://user-images.githubusercontent.com/6243418/118151033-7c6f9d80-b413-11eb-8700-f0a87bc786b5.JPG)

Please could you tell me where I'm wrong, or post an example for the "volumes" part of the docker-compose.yml file ?
Thanks a lot ...

**Comment by lastzero** [comment, marked as answer]
I assume you've been using [our docker-compose.yml example for Windows](https://dl.photoprism.org/docker/windows/docker-compose.yml)? Mounting drives other than C: should work like this when WSL is disabled for Docker:

```yaml
- "P:/:/photoprism/originals"
```

Alternatively, you may try to mount the drive / folder via Docker as shown here:

https://docs.photoprism.org/getting-started/faq/#how-can-i-mount-network-shares-with-docker

**Reply by cedrik72** [reply]
Hi, 
Thanks a lot for reply.

The line you told is exactly the line I add to docker-compose witch is the Windows example.
All seems ok there.
But is the error about mounting the drive ?
As I understand on the error line it seems that PP (or Docker) doesn't have rights to "create mount source path" 

     ERROR: for photoprism  Cannot start service photoprism: error while creating mount source path '/host_mnt/uC/SERVCBAD/Photos': mkdir /host_mnt/uC: operation not permitted

I tried to mount like that 

```
volumes:
  photoprism-originals:
    driver: local
    driver_opts:
      type: cifs
      o: "username=user,password=secret,rw"
      device: "//host/folder" 
```
But I'm not successful with this, I'm not sure I put it the good way !
Do I add the lines **after** the Volumes: in the docker-compose file, or do I add an other Volumes: entry ?

When I try, every ways gave me:
```
C:\Users\SON\Desktop\photoprism conf>docker-compose up -d
ERROR: yaml.parser.ParserError: while parsing a block mapping
  in ".\docker-compose.yml", line 81, column 7
expected <block end>, but found '-'
  in ".\docker-compose.yml", line 86, column 7
```
or
```
C:\Users\SON\Desktop\photoprism conf>docker-compose up -d
ERROR: The Compose file '.\docker-compose.yml' is invalid because:
Unsupported config option for services.photoprism: 'photoprism-originals'
```
(I have filled in user and pass for the login to mount drive login)

Sorry but I cant manage to make it work for a network drive :-(

I tested PP with local drive and it's really good 👍 
This will finished with a new HDD plugged direct on the NUC , then it'll be a real local drive, much easyer.

**Reply by bencollinz** [reply]
Having the same problem. Not sure why this marked solved. Did you figure it out?

**Reply by lastzero** [reply]
What problem? How does your config look like? Did you compare it with our example and read the docs? What did you try to solve it?

**Comment by imiMike** [comment]
Have you figured it out? It doesn’t work for me either with a mapped network drive. While indexing it just doesn’t see any files. I’ve added this to the yml file: - "P:/:/photoprism/originals" .
Any ideas?
I’m using a WD Cloud network drive with smb.
